src/components/organisms/topbar/project/navigation.tsx

```diff
@@ -45,9 +47,14 @@ export const ProjectTopbarNavigation = () => {
 				const getPath = () => {
 					switch (item.key) {
 						case "assets":
-							return latestOpenedTab ? `/${latestOpenedTab}` : "/code";
+							return latestOpened.tab ? `/${latestOpened.tab}` : "/code";
 						case "sessions":
-							return deploymentId ? `/deployments/${deploymentId}/sessions` : "";
+							return latestOpened.sessionId
```

Comment:
if open the session page from the deployments page, latestOpened.sessionId reset

Reply:
refactored here: https://github.com/autokitteh/web-platform/pull/732/commits/a2b9251aa741bd883b6d9665453d34903cee84e6